### ISSUE TITLE
No podcast PodcastService.php error

### ISSUE THREAD

**Issue by momai** [LibreTime/libretime#485]
Hello! I installed LibreTime alpha-3 on debian 9 and I have a problem with going to feeds/station-rss, I get an error in the browser. Failed to load resource: the server responded with a status of 500 (Internal Server Error).

logfile zendphp.log, I see an error:

 `[ErrorController.php:54 - errorAction()] - An internal application error has occurred.: ErrorException: DOMDocument::loadXML(): StartTag: invalid element name in Entity,
 line: 2 in /usr/share/airtime/php/airtime_mvc/application/services/PodcastService.php:517
Stack trace:
#0 [internal function]: exception_error_handler(2, 'DOMDocument::lo...', '/usr/share/airt...', 517, Array)
#1 /usr/share/airtime/php/airtime_mvc/application/services/PodcastService.php(517): DOMDocument->loadXML('<?xml version="...')
#2 /usr/share/airtime/php/airtime_mvc/application/controllers/FeedsController.php(19): Application_Service_PodcastService::createStationRssFeed()
#3 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Action.php(516): FeedsController->stationRssAction()
#4 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Dispatcher/Standard.php(308): Zend_Controller_Action->dispatch('stationRssActio...')
#5 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#6 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application/Bootstrap/Bootstrap.php(105): Zend_Controller_Front->dispatch()
#7 /usr/share/airtime/php/vendor/zendframework/zendframework1/library/Zend/Application.php(384): Zend_Application_Bootstrap_Bootstrap->run()
#8 /usr/share/airtime/php/airtime_mvc/application/airtime-boot.php(84): Zend_Application->run()
#9 /usr/share/airtime/php/airtime_mvc/public/index.php(68): require_once('/usr/share/airt...')
#10 {main}
`

Help please solve the problem.

**Comment by hairmare** [comment]
It looks like the podcast isn't returning a valid RSS document. Can you share the URL to the feed?

**Comment by momai** [comment]
I unfortunately retired from affairs related to LibreTite. However, what information would you like to receive? I could take from a virtual machine and provide you with what you ask. If this is a necessity for you. For me personally, eliminating the problem is no longer relevant ...

**Comment by Robbt** [comment]
Thanks for the feedback @momai no other work is necessary, thanks for responding. We will just use this as a reminder to test the podcast publishing for basic functionality and close it if we can't duplicate the error you ran into.

**Comment by ciaby** [comment]
I fixed it by changing this piece of code:
```
    private static function addEscapedChild($node, $name, $value = null, $namespace = null) {
        if (empty($value)) {
            return null;
        }
        $child = $node->addChild($name, $value, $namespace);
        //$child->{0} = $value;
        return $child;
```
the value of $child should be passed directly to addChild and not as a separate node {0}
Works for me :)

**Comment by Robbt** [comment]
Cool thanks for the solution would you mind submitting a PR with your fix so we can test and apply it ?

**Comment by paddatrapper** [comment]
Fixed in #810
